### ISSUE TITLE
Plugin can't handle csv files with more than 22 rows of data

### ISSUE THREAD

**Issue by prototype99** [deathau/csv-obsidian#3]
plugin version: 0.01
obsidian version: 0.11.13
OS: Linux
file in question: https://raw.githubusercontent.com/prototype99/price-razor/master/data.csv
related to: #5
issue: only loads 22 lines from the file out of a total 341 lines.

otherwise, nothing seems particularly off, but of course, that harms being able to use it in a real-world context. Making a CSV "from scratch" does not fix anything. I'm not quite sure if you'd want me to put issues experienced while trying to fix this issue as separate issues in case they are actually the same issue, so any additional issues experienced will be put below.

**Comment by prototype99** [comment]
plugin cannot handle empty files, will attempt loading a file with just the headers

**Comment by prototype99** [comment]
had to add at least one line with data before it could parse. I inserted rows and it seems to be unable to insert any rows past row 22. The true issue seems to be that the row limit is 22, so I'll change the title to match

**Comment by twilightfades0** [comment]
I just installed this plugin and opened a 176 line csv no problem, fwiw!

**Comment by prototype99** [comment]
hmm what does the file you used look like? prompted by this I tried again and still had the same issue

**Comment by prototype99** [comment]
alternatively: did you compile from source or something?

**Comment by twilightfades0** [comment]
Sorry for the delay!! Um, what do you mean? 😅 I don't know how to do much of anything myself so I just did what It said to do, I didn't alter anything. I'm using the most up to date release of obsidian, are you? (that's just the only thing i can think of!)

**Comment by prototype99** [comment]
It's okay! I've been busy anyway. Yeah I'm using 0.11.13 which is the latest release, I installed it through the built-in plugin store. I wasn't quite sure what you meant so I looked under installation and tried both "from github" and "development" but nothing changed, so there must be some sort of difference. Are you also using Linux?

**Comment by calvinhobbes23** [comment]
I am sorry to say this plugin is useless because of this issue. It does not display more that 10-20 rows. Don't waste your time. Saying this because I am wondering how much time it takes to solve a critical bug reported more than a year ago!

**Comment by swehba** [comment]
@calvinhobbes23 I agree. Just tried this plugin for the first time, and could not open a CSV with around 100 rows. Not useful in its present state.
